package.json

```diff
@@ -6,9 +6,9 @@
     "@testing-library/jest-dom": "^4.2.4",
     "@testing-library/react": "^9.3.2",
     "@testing-library/user-event": "^7.1.2",
-    "react": "^16.14.0",
-    "react-dom": "^16.14.0",
-    "react-router-dom": "^5.2.0",
+    "react": "^18.2.0",
+    "react-dom": "^18.2.0",
+    "react-router-dom": "^6.11.2",
     "react-scripts": "^3.4.4"
```

Comment:
I'm not sure if this would break anything else - but I can't run this project without also updating the react-scripts version. It's giving an error that suggests that my current version of node (most recent version) doesn't support this version of react-scripts. I don't know if it's just an issue on my end, but I assume it will be an issue for any student using modern Node. 

If I update to the most recent version (5 something) then everything works as expected.

Reply:
Thank you!  That's super helpful.  I'll make that update

Reply:
Thank you!  That's super helpful.  I'll make that update